packages/e2e-test-utils-playwright/src/editor/click-block-appender.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Clicks the default block appender.
+ *
+ * @param {Editor} this
+ */
+export async function clickBlockAppender( this: Editor ) {
```

Comment:
I don't think we need to introduce this util. We can simply just do `await this.page.click( 'role=button[name="Add default block"i]' )` in most cases inline. In some rare cases where clearing selected blocks is necessary, we can explicitly do that to avoid confusion?